sensors/test_deps.go

```diff
@@ -130,7 +130,7 @@ func getInvalidLidar() *inject.Camera {
 		return nil, transform.NewNoIntrinsicsError("")
 	}
 	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
-		return camera.Properties{}, nil
+		return camera.Properties{SupportsPCD: true}, nil
```

Comment:
probably ok to leave SupportsPCD out here, since this is for an invalid lidar

Reply:
I think since we are checking that it supports the properties upon creation of the new lidar, we do need SupportsPCD to be true here, because otherwise we would catch the unsupported properties before we get to the erroring functions